docs/userguide/quickstart.rst

```diff
@@ -56,6 +56,14 @@ containing a ``build-system`` section similar to the example below:
 This section declares what are your build system dependencies, and which
 library will be used to actually do the packaging.
 
+.. note::
+
+   Historically this documentation has wrongly listed ``wheel``
+   in the ``requires`` list, and many projects still do that. This is
+   incorrect. The backend automatically adds ``wheel`` dependency
+   when it is required, and listing it explicitly causes it to be
+   unnecessarily required for source distribution builds.
```

Comment:
```suggestion
   Historically this documentation has unnecessarily listed ``wheel``
   in the ``requires`` list, and many projects still do that. This is
   not recommended. The backend automatically adds ``wheel`` dependency
   when it is required, and listing it explicitly causes it to be
   unnecessarily required for source distribution builds.
   You should only explicitly list `wheel` if you are importing it
   in your `setup.py`.
```

Reply:
Thanks. I've merged your change but fixed formatting and changed "access to it" to "access it" that I believe is more grammatically correct.